controllers/taskcluster_operations.go

```diff
@@ -664,14 +664,14 @@ func (o *TaskClusterOperations) RenderValues(ctx context.Context) (*TaskClusterV
 			return nil, err
 		}
 
-		var webhookSecrets []string
+		webhookSecret := ""
 		for k, v := range secret.Data {
 			if !strings.HasPrefix("webhook-secret", k) {
 				continue
 			}
 
 			s := (string)(v)
-			webhookSecrets = append(webhookSecrets, s)
+			webhookSecret += fmt.Sprintf("'%s'", s)
```

Comment:
Just so this doesn't cost you any sanity points, expected produced values look like: `"value 1"'value 2'`. This is a bit crazy but I've tested it and it does work good.

Reply:
Does this need a note in the docs regarding the format?

Reply:
That would be a good shout I think, I had to produce that example by reverse-engineering the regular expression. 😄